Gruntfile.js

```diff
@@ -51,6 +52,33 @@ module.exports = function(grunt) {
                     'build/hm-client.min.js': ['<%= concat.dist.dest %>']
                 }
             }
+        },
+        less: {
+          development: {
+            files: {
+              'build/styles.css': 'app/components/**/*.less'
```

Comment:
why does development create a file in `build` ? 
wouldn't it be better to create a `tmp` folder or something?